clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -345,6 +345,9 @@ public void onFailure(RuntimeException e) {
                 cb.onComplete(offsets, e);
             }
         });
+
+        // ensure commit has a chance to be transmitted (without blocking on its completion)
+        client.poll(0);
```

Comment:
Do we expect things like `WakeupException` to be acceptable here? Should we use `client.quickPoll()` instead?